content/soarian/r4/clinical/medications/immunization.md

```diff
@@ -0,0 +1,171 @@
+* [Administration date](https://hl7.org/fhir/R4/immunization-definitions.html#Immunization.occurrence_x_){:target="_blank"}
+* [Indicates if the record is from the source who administered the vaccine](https://hl7.org/fhir/R4/immunization-definitions.html#Immunization.primarySource){:target="_blank"}
+
+
+## Terminology Bindings
+
+<%= terminology_table(:soarian_immunization, :r4) %>
+
+## Search
+
+Search for Immunization resources that meet the specified query parameters:
+
+    GET /Immunization?:parameters
+
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true)%>
+
+### Parameters
+
+ Name          | Required?                                    | Type                                                       | Description
+---------------|----------------------------------------------|------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------
+ `_id`         | This, or `patient`                            | [`token`](https://hl7.org/fhir/R4/search.html#token)        | The logical resource ID associated with the resource.
+ `patient`	   | This, or `_id`								   | [`reference`](https://hl7.org/fhir/r4/search.html#reference)| The patient for the vaccination record. 
+ `_revinclude` | No 										   | [`_revinclude`](https://hl7.org/fhir/search.html#revinclude)| A request to include any Provenance resource in the bundle that refers to a Immunization resource in the search results. Only supported with Provenance.
+
+
+
+### Headers
+
+<%= headers fhir_json: true %>
+
+### Example Search by Patient
+
+#### Request
+
+    GET https://fhir-myrecord-sc.sandboxcerner.com/r4/3f2aca24-87f3-4eac-a6d7-1f75247e6b43/Immunization?patient=4C05F5F357EE44FC874C0CF4AB249F99
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:SOARIAN_R4_IMMUNIZATION_SEARCH_BY_PT) %>
+
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
+
+### Example Search by ID
+
+#### Request
+
+	GET https://fhir-myrecord-sc.sandboxcerner.com/r4/3f2aca24-87f3-4eac-a6d7-1f75247e6b43/Immunization?_id=4C05F5F357EE44FC874C0CF4AB249F99.000001001008000061001&patient=4C05F5F357EE44FC874C0CF4AB249F99
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:SOARIAN_R4_IMMUNIZATION_SEARCH_BY_ID) %>
+
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Retrieve by ID
+
+List an individual Immunization resource by its ID:
+
+    GET /Immunization/:id
+
+## Authorization Types
+
+<%= authorization_types(provider: true, patient: true)%>
+
+### Headers
+
+<%= headers fhir_json: true %>
```

Comment:
Authorization: <OAuth2 Bearer Token> not needed?

Reply:
Refer screenshot above .OAuth bearer token is included for authorization in the header

Reply:
Then why is that missing in the code? Just wondering